docs/providers/README.md

```diff
@@ -57,6 +57,13 @@ Under the hood, the Serverless Framework is deploying your code to a cloud provi
     </div>
   </div>
   </a>
+  <a href="./knative/">
+  <div class="providerSection">
+    <div class="providerSectionHeader">
+        <img src="https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/images/docs/knative-logo.svg" width="250" draggable="false" class='knative-logo'/>
```

Comment:
@pdaryani This is something we need to setup / change (I just followed the patter of the other documentations)

Reply:
@pmuens logo has been updated :)

Reply:
Great! Thanks a lot @pdaryani :+1: